CleanWeather/Support/SuperViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SuperViewController.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 16/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+class SuperViewController: UIViewController {
+    
+    func presentSpinner(_ state: Bool) {
+        let spinner = SpinnerController()
```

Comment:
nie znika Ci spinner, bo ty jak wywolasz presentSpinner(false) robisz tutaj nowy spinner lokalnie i probujeesz go usunac z kontrollera (go tam nie bylo, tam jest inny spinner)

to powinna byc zmienna na poziomie kontrolera

Reply:
Aaaaaaa. No jasne. 👍